### ISSUE TITLE
Error running releasify, could not load candle

### ISSUE THREAD

**Issue by davetimmins** [Squirrel/Squirrel.Windows#468]
Hi,

I'm running on windows 10 with version 1.1.1 of squirrel and I get the following exception when calling releasify

```
2015-10-16 09:03:52> Program: Starting Squirrel Updater: --releasify C:\code\projects\myapp\artifacts\packs\myapp\myapp.1.0.0.nupkg -r C:\code\projects\myapp\artifacts\myappRelease
2015-10-16 09:03:52> Program: Bootstrapper EXE found at:C:\code\projects\myapp\packages\FAKE\squirrel.windows\tools\Setup.exe
2015-10-16 09:03:52> Program: Creating release package: C:\code\projects\myapp\artifacts\myappRelease\myapp.1.0.0.nupkg
2015-10-16 09:03:52> ReleasePackage: Creating release package: C:\code\projects\myapp\artifacts\myappRelease\myapp.1.0.0.nupkg => C:\code\projects\myapp\artifacts\myappRelease\myapp-1.0.0-full.nupkg
2015-10-16 09:03:53> ReleasePackage: Extracting dependent packages: []
2015-10-16 09:03:53> ReleasePackage: Removing unnecessary data
2015-10-16 09:03:54> Program: Building embedded zip file for Setup.exe
2015-10-16 09:03:56> Unhandled exception: System.AggregateException: One or more errors occurred. ---> System.Exception: Failed to compile WiX template, command invoked was: 'candle.exe -nologo -ext WixNetFxExtension -out "C:\code\projects\myapp\artifacts\myappRelease\Setup.wixobj" "C:\code\projects\myapp\artifacts\myappRelease\Setup.wxs"'

Output was:
Unhandled Exception: System.IO.FileLoadException: Could not load file or assembly 'candle, Version=3.0.0.0, Culture=neutral, PublicKeyToken=36e4ce08b8ecfb17' or one of its dependencies. Strong name validation failed. (Exception from HRESULT: 0x8013141A) ---> System.Security.SecurityException: Strong name validation failed. (Exception from HRESULT: 0x8013141A)
   --- End of inner exception stack trace ---
   at Squirrel.Update.Program.<createMsiPackage>d__ad.MoveNext()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Squirrel.Update.Program.Releasify(String package, String targetDir, String packagesDir, String bootstrapperExe, String backgroundGif, String signingOpts, String baseUrl, String setupIcon, Boolean generateMsi)
   at Squirrel.Update.Program.executeCommandLine(String[] args)
   at Squirrel.Update.Program.main(String[] args)
---> (Inner Exception #0) System.Exception: Failed to compile WiX template, command invoked was: 'candle.exe -nologo -ext WixNetFxExtension -out "C:\code\projects\myapp\artifacts\myappRelease\Setup.wixobj" "C:\code\projects\myapp\artifacts\myappRelease\Setup.wxs"'

Output was:
Unhandled Exception: System.IO.FileLoadException: Could not load file or assembly 'candle, Version=3.0.0.0, Culture=neutral, PublicKeyToken=36e4ce08b8ecfb17' or one of its dependencies. Strong name validation failed. (Exception from HRESULT: 0x8013141A) ---> System.Security.SecurityException: Strong name validation failed. (Exception from HRESULT: 0x8013141A)
   --- End of inner exception stack trace ---
   at Squirrel.Update.Program.<createMsiPackage>d__ad.MoveNext()<---
```

if I run it with the `--no-msi` option it runs fine. The log for that is

```
2015-10-16 09:03:02> Program: Starting Squirrel Updater: --releasify C:\code\projects\myapp\artifacts\packs\myapp\myapp.1.0.0.nupkg --no-msi -r C:\code\projects\myapp\artifacts\myappRelease
2015-10-16 09:03:02> Program: Bootstrapper EXE found at:C:\code\projects\myapp\packages\FAKE\squirrel.windows\tools\Setup.exe
2015-10-16 09:03:02> Program: Creating release package: C:\code\projects\myapp\artifacts\myappRelease\myapp.1.0.0.nupkg
2015-10-16 09:03:02> ReleasePackage: Creating release package: C:\code\projects\myapp\artifacts\myappRelease\myapp.1.0.0.nupkg => C:\code\projects\myapp\artifacts\myappRelease\myapp-1.0.0-full.nupkg
2015-10-16 09:03:02> ReleasePackage: Extracting dependent packages: []
2015-10-16 09:03:02> ReleasePackage: Removing unnecessary data
2015-10-16 09:03:03> Program: Building embedded zip file for Setup.exe
```
any suggestions as to what I can try for more information?

**Comment by davetimmins** [comment]
Aaaaaaaaaaaaaaaaaaaand you've already fixed it in v1.1.2. Legend!

**Comment by paulcbetts** [comment]
@davetimmins Fuck strong naming forever
